dotnet-desktop-guide/net/winforms/data/overview.md

```diff
@@ -0,0 +1,250 @@
+
+- <xref:System.ComponentModel.IBindingList> interface
+
+  A class that implements the <xref:System.ComponentModel.IBindingList> interface provides a much higher level of data-binding functionality. This implementation offers you basic sorting capabilities and change notification. Both are useful when the list items change, and when the list itself changes. Change notification is important if you plan to have multiple controls bound to the same data. It helps you to make data changes made in one of the controls to propagate to the other bound controls.
+
+  > [!NOTE]
+  > Change notification is enabled for the <xref:System.ComponentModel.IBindingList> interface through the <xref:System.ComponentModel.IBindingList.SupportsChangeNotification%2A> property which, when `true`, raises a <xref:System.ComponentModel.IBindingList.ListChanged> event, indicating the list changed or an item in the list changed.
+
+  The type of change is described by the <xref:System.ComponentModel.ListChangedType> property of the <xref:System.ComponentModel.ListChangedEventArgs> parameter. Hence, whenever the data model is updated, any dependent views, such as other controls bound to the same data source, will also be updated. However, objects contained within the list will have to notify the list when they change so that the list can raise the <xref:System.ComponentModel.IBindingList.ListChanged> event.
+
+  > [!NOTE]
+  > The <xref:System.ComponentModel.BindingList%601> provides a generic implementation of the <xref:System.ComponentModel.IBindingList> interface.
+
+- <xref:System.ComponentModel.IBindingListView> interface
+
+  A class that implements the <xref:System.ComponentModel.IBindingListView> interface provides all the functionality of an implementation of <xref:System.ComponentModel.IBindingList>, along with filtering and advanced sorting functionality. This implementation offers string-based filtering, and multi-column sorting with property descriptor-direction pairs.
+
+- <xref:System.ComponentModel.IEditableObject> interface
+
+  A class that implements the <xref:System.ComponentModel.IEditableObject> interface allows an object to control when changes to that object are made permanent. This implementation supports the <xref:System.ComponentModel.IEditableObject.BeginEdit%2A>, <xref:System.ComponentModel.IEditableObject.EndEdit%2A>, and <xref:System.ComponentModel.IEditableObject.CancelEdit%2A> methods, which enable you to roll back changes made to the object. Following is a brief explanation of the functioning of the <xref:System.ComponentModel.IEditableObject.BeginEdit%2A>, <xref:System.ComponentModel.IEditableObject.EndEdit%2A>, and <xref:System.ComponentModel.IEditableObject.CancelEdit%2A> methods and how they work with one another to enable a possible rollback of changes made to the data:
```

Comment:
Can you clean up all of the xref links in the article paragraphs? There are just too many and it's a bit distracting to read. all of the old windows forms content is like this, yuck!

Within the same paragraph once the first XREF of a type has been used, just change the next mention to `inline code`. For example in this paragraph:

```suggestion
  A class that implements the <xref:System.ComponentModel.IEditableObject> interface allows an object to control when changes to that object are made permanent. This implementation supports the <xref:System.ComponentModel.IEditableObject.BeginEdit%2A>, <xref:System.ComponentModel.IEditableObject.EndEdit%2A>, and <xref:System.ComponentModel.IEditableObject.CancelEdit%2A> methods, which enable you to roll back changes made to the object. Following is a brief explanation of the functioning of the `BeginEdit`, `EndEdit`, and `CancelEdit` methods and how they work with one another to enable a possible rollback of changes made to the data:
```

The next paragraph is even worse. This is how it looks rendered; just too many links

![image](https://user-images.githubusercontent.com/67293991/168643170-ba53cf45-f2ed-426b-92da-b2f78aaf217d.png)

Reply:
Ok